x-pack/test/security_solution_cypress/cypress/e2e/explore/cases/creation.cy.ts

```diff
@@ -48,16 +48,21 @@ import {
   fillCasesMandatoryfields,
   filterStatusOpen,
 } from '../../../tasks/create_new_case';
-import { login } from '../../../tasks/login';
 import { visit, visitWithTimeRange } from '../../../tasks/navigation';
 
 import { CASES_URL, OVERVIEW_URL } from '../../../urls/navigation';
 import { ELASTICSEARCH_USERNAME, IS_SERVERLESS } from '../../../env_var_names_constants';
 import { deleteCases } from '../../../tasks/api_calls/cases';
+import { login } from '../../../tasks/login';
 
-// https://github.com/elastic/kibana/issues/179231
 const isServerless = Cypress.env(IS_SERVERLESS);
-const username = isServerless ? 'platform_engineer' : Cypress.env(ELASTICSEARCH_USERNAME);
+const getUsername = () => {
+  if (isServerless) {
+    return cy.task('getFullname');
```

Comment:
I am not familiar with this part, but curious why do we get the full name and not the username. Is it not available in Serverless?

Reply:
I was using code that was already available so I don't know which is the correct answer for that.